python_functionality/homework.py

```diff
@@ -11,45 +11,56 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
     Make all `names` field in list of students to start from upper letter
 
     Examples:
-        fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
+        fix_names_start_letters([{'name': 'Alex', 'age': 26},
+        {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for k in data:
+        if k.get('name'):
+            k['name'] = k['name'].capitalize()
+    return data
 
 
 def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
     """given_data
     Remove from dictionaries given key value
 
     Examples:
-       remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
+       remove_dict_field([{'name': 'Alex', 'age': 26},
+       {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+
+    for i in data:
+        for j in redundant_keys:
+            del i[j]
+    return data
 
 
 def task_3_find_item_via_value(data: DT, value) -> DT:
     """
     Find and return all items that has @searching value in any key
     Examples:
-        find_item_via_value([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 26)
+        find_item_via_value([{'name': 'Alex', 'age': 26},
+        {'name': 'denys', 'age': 89}], 26)
         >>> [{'name': 'Alex', 'age': 26}]
     """
-    pass
+
+    return [dict for dict in data if value in dict.values()]
```

Comment:
Do not use python reserved words like`list, dict, def, class, etc` for naming variables